### PR TITLE
SHS-60: Align view “more link” buttons to the right

### PR DESCRIPTION
**Pull request by callinmullaney**
# READY FOR REVIEW
https://fourkitchens.atlassian.net/browse/SHS-60 

## Summary
Aligns all "more links" added by the ViewUI to the right-hand side of the container

## Need Review By (Date)
unknown

## Urgency
low

## Steps to Test
1. Pull code, start your local lando/gitpod instance. 
2. Go to https://hs-colorful.suhumsci.loc/ (homepage of hs-colorful sight) and verify the "more link" for the `Spotlight` news section is aligned to the right of its container.
3. Go to https://hs-colorful.suhumsci.loc/default-views/news (default views page for hs-colorful) and verify the "more link" for the `Spotlight` news section is aligned to the right of its container.
4. Review the code changes to `humsci_basic` and verify they are global enough to affect the other sites.

### CODE REVIEW

**Comment by su-sws-jira** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1155-bs0jg2rekbdqapfwb0rz40vywf9rp2ll.tugboatqa.com
- https://hs-colorful-bs0jg2rekbdqapfwb0rz40vywf9rp2ll.tugboatqa.com
- https://hs-traditional-bs0jg2rekbdqapfwb0rz40vywf9rp2ll.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62ebda0b7e6c3f16daafd1f7

File: docroot/themes/humsci/humsci_basic/src/scss/components/_buttons.scss [hunk 1]
```diff
@@ -70,6 +70,9 @@ button.more-link,
 // This adds margin to the top of more link which is needed for hb-grid views
 .more-link {
   .views-element-container & {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: flex-end;
     margin-top: hb-calculate-rems(24px);
```
Comment (ryanhagerty) [inline docroot/themes/humsci/humsci_basic/src/scss/components/_buttons.scss:76]: We'll also need to add the other compiled CSS files, not just colorful.

**Comment by ryanhagerty** [comment]
@callinmullaney 

I'm not seeing a change to the Spotlight sections on these two pages. I recompiled the CSS and cleared caches. 

Here's some screenshots (images started being broken on Gitpod a couple weeks ago):
Screen Shot 2022-08-08 at 8 52 33 AM

Screen Shot 2022-08-08 at 8 54 04 AM

Let me know if I'm just looking at the wrong component, but everything looks the same as the test site: https://hs-colorful.stanford.edu/

One more thing: let's change the base branch from `develop` to the release branch `fk-stnfd-sprint-7`. This will allow us to review and merge ourselves and then near release day we can make a PR for `fk-stnfd-sprint-7` to go into `develop`. This will also reduce the need to bug Mike about merging PRs.

**Comment by su-sws-jira** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1155-bs0jg2rekbdqapfwb0rz40vywf9rp2ll.tugboatqa.com
- https://hs-colorful-bs0jg2rekbdqapfwb0rz40vywf9rp2ll.tugboatqa.com
- https://hs-traditional-bs0jg2rekbdqapfwb0rz40vywf9rp2ll.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62ebda0b7e6c3f16daafd1f7

**Comment by callinmullaney** [comment]
@ryanhagerty The styles/links in the screenshots above are not `.more-link` 's as they do not have this class. According to the acceptance criteria on the ticket 
`This work only impacts the “more-link” class and DOES NOT impact the “hs-more-link” styles`

I updated the changes in this PR to affect all `.more-link` classes and not just ones added to a view. If we/client want these other node specific links to also be right-aligned we'll need to add `.more-link` to these various node templates or via a preprocessing function. 
Below are screenshots from the Music and Economics sites for testing:
Screenshot from 2022-08-09 09-28-34
Screenshot from 2022-08-09 09-28-23

**Review by ryanhagerty — APPROVED** [review]
approved whee 🍔

File: docroot/themes/humsci/humsci_basic/src/scss/components/_buttons.scss (hunk 1 above)
Comment (callinmullaney) [reply docroot/themes/humsci/humsci_basic/src/scss/components/_buttons.scss:76]: @ryanhagerty are there other compiled files besides `humsci_airy/css/humsci_airy.css`, `humsci_colorful/css/humsci_colorful.css`, and `humsci_traditional/css/humsci_traditional.css` I should be including?

**Comment by su-sws-jira** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Links:
- https://pr1155-bs0jg2rekbdqapfwb0rz40vywf9rp2ll.tugboatqa.com
- https://hs-colorful-bs0jg2rekbdqapfwb0rz40vywf9rp2ll.tugboatqa.com
- https://hs-traditional-bs0jg2rekbdqapfwb0rz40vywf9rp2ll.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/62ebda0b7e6c3f16daafd1f7

File: docroot/themes/humsci/humsci_basic/src/scss/components/_buttons.scss (hunk 1 above)
Comment (ryanhagerty) [reply docroot/themes/humsci/humsci_basic/src/scss/components/_buttons.scss:76]: There aren't - just as a head's up the `humsci_airy` is considered abandoned, but we still include it.

**Pull request merged**